app/schemas/students/response/timetable.py

```
@@ -0,0 +1,17 @@
+from typing import List
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)